src-gui/src/renderer/components/modal/seed-selection/SeedSelectionDialog.tsx

```diff
@@ -250,23 +278,44 @@ export default function SeedSelectionDialog() {
         )}
 
         {selectedOption === "FromSeed" && (
-          <TextField
-            fullWidth
-            multiline
-            rows={3}
-            label="Enter your seed phrase"
-            value={customSeed}
-            onChange={(e) => setCustomSeed(e.target.value)}
-            placeholder="Enter your Monero 25 words seed phrase..."
-            error={!isSeedValid && customSeed.length > 0}
-            helperText={
-              isSeedValid
-                ? "Seed is valid"
-                : customSeed.length > 0
-                  ? "Seed is invalid"
-                  : ""
-            }
-          />
+          <>
+            <TextField
+              fullWidth
+              multiline
+              rows={3}
+              label="Enter your seed phrase"
+              value={customSeed}
+              onChange={(e) => setCustomSeed(e.target.value)}
+              placeholder="Enter your Monero 25 words seed phrase..."
+              error={!isSeedValid && customSeed.length > 0}
+              helperText={
+                isSeedValid
+                  ? "Seed is valid"
+                  : customSeed.length > 0
+                    ? "Seed is invalid"
+                    : ""
+              }
+            />
+            <TextField
+              type="text"
+              inputProps={{
+                inputmode: "numeric",
+                pattern: "[0-9]*",
+              }}
+              label="Restore blockheight (optional)"
+              value={blockheightInput}
+              onChange={(e) => setBlockheight(e.target.value)}
+              placeholder="Restore blockheight (optional)"
```

Comment:
Change to something more descriptive like "Restore blockheight, leave empty to scan from start of the Blockchain".